crates/core/tedge/src/system_services/managers/mod.rs

```diff
@@ -1,6 +1,4 @@
-mod bsd;
-mod null;
-mod openrc;
-mod systemd;
```

Comment:
The corresponding `.rs` files must be removed and their content moved in `configuration/contrib/system/*/system.toml ` configuration files.

Reply:
Done. Added `system.toml` for OpenRC and BSD as well as systemd.